src/components/Utils/EnterpriseBox.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <KAlert
+    class="py-1"
+    appearance="success"
+    size="small"
+  >
+    <template slot="alertMessage">
+      <div class="alert-content flex items-center text-sm">
+        <div class="mr-3">
+          Do more with Kuma <span class="alert-content__arrow">&rarr;</span>
+        </div>
+        <div>
+          <KButton
+            class="no-underline"
+            appearance="primary"
+            size="small"
+            target="_blank"
+            :to="url"
+          >
+            Enterprise
+          </KButton>
+        </div>
+      </div>
+    </template>
+  </KAlert>
+</template>
+
+<script>
+export default {
+  name: 'EnterpriseBox',
+  data () {
+    return {
+      url: 'https://kuma.io/enterprise/'
```

Comment:
@bartsmykla Changed this to the [Kuma Enterprise page](https://kuma.io/enterprise/).